spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -76,10 +78,11 @@
         user_id: user.id,
         threatmetrix_session_id: threatmetrix_session_id,
         request_ip: request_ip,
+        issuer: issuer,
       )
     end
 
-    context 'webmock lexisnexis and threatmetrix' do
+    context 'webmock lexisnexis and threatmetrix is on for service provider' do
```

Comment:
did this to make the diff clear that not much was changed (ie not another context)

Reply:
I'd consider doing the context. We can add the "hide whitespace" options when reviewing to help diff look manageable. The current state has a lot of duplicated setup code is it's still a fairly noisy diff

Reply:
ah ok.